src/components/accounts/AccountFormAccountId.js

```diff
@@ -109,9 +109,6 @@ class AccountFormAccountId extends Component {
         
         handleChange(e, { name, value })
 
-        !this.props.formLoader && this.checkAccountIdLength(value) && this.props.setFormLoader(true)
-        this.props.formLoader && !this.checkAccountIdLength(value) && this.props.setFormLoader(false)
```

Comment:
@vgrichina this is necessary - without that `checking...` does not appear correctly, if we don't want to use it we need to figure out some replacement. I can handle it after #1108  will be merged.